ui.js

```diff
@@ -149,7 +154,7 @@ class Emoj extends Component {
 					return;
 				}
 
-				process.stdin.on('keypress', this.handleKeyPress);
+				this.props.stdin.on('data', this.handleInput);
```

Comment:
Why the switch from `keypress` to `data` event?

Reply:
Ink no longer uses `readline.emitKeypressEvents`, just raw `setRawMode(true)` + `data` events.